src/StarsRatingBar/index.d.ts

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+export interface StarsRatingBarProps {
+  dataHook?: string;
+  className?: string;
+  size?: starRatingBarSize;
+  readOnly?: boolean;
+  rateCaptions?: string[];
```

Comment:
This is more specific

```suggestion
  rateCaptions?: [string, string, string, string, string];
```